torch/csrc/cuda/nccl.cpp

```diff
@@ -99,6 +96,13 @@ ncclDataType_t to_nccl_data_type(const at::Tensor& t) {
   }
```

Comment:
nit: this is prior to this PR, shall we also tell the type in the error message?

Reply:
fixed